### PR TITLE
colw/Add link to buy Ledger Nano S in about page.

### PR DESCRIPTION
**Pull request by colw**
**Description:**

Add link to buy ledger nano S

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/PageAbout.vue [hunk 1]
```diff
@@ -43,7 +43,12 @@
       <p>
         To send transactions with Lunie, you'll have to sign them with your
         Ledger Nano&nbsp;S. If you don't have a Ledger Nano S, you can
-        <a href="" target="_blank" rel="noopener norefferer">buy one here</a>.
+        <a
+          href="https://shop.ledger.com/products/ledger-nano-s"
```
Comment (jbibla) [inline src/components/common/PageAbout.vue]:
```suggestion
          href="https://shop.ledger.com/?r=3dd204ef7508"
```

**Comment by jbibla** [comment]
@colw can you add the same link to these session screens?

"Don't have a Ledger yet? _Get one here_."

Screen Shot 2019-05-27 at 11 25 33 AM
Screen Shot 2019-05-27 at 11 25 41 AM

File: src/components/common/PageAbout.vue (hunk 1 above)
Comment (colw) [reply src/components/common/PageAbout.vue]: Should we just refer to the generic 'Ledger', or 'Ledger Nano`?

**Comment by colw** [comment]
Top screen: Do you want it in addition to the ledger app message?

File: src/components/common/PageAbout.vue (hunk 1 above)
Comment (jbibla) [reply src/components/common/PageAbout.vue]: i think we can say "Ledger Nano" for now on - since we support S and X - what do you think?

Comment (colw) [reply src/components/common/PageAbout.vue]: Sure 👍

**Comment by colw** [comment]
Screenshot 2019-05-29 at 16 40 45

Screenshot 2019-05-29 at 16 40 54

I also removed all references to `Nano S` in favour of `Nano`.

**Comment by colw** [comment]
I could also remove he `max-width: 320px` on that top label?

**Pull request merged**